edxval/tests/test_api.py

```diff
@@ -269,6 +275,94 @@ def test_get_url_for_profile(self):
         self.assertEqual(url, u'http://www.meowmix.com')
 
 
+class GetVideoForCourseProfiles(TestCase):
+    """Tests get_video_info_for_course_and_profiles in api.py"""
+
+    def setUp(self):
+        """
+        Creates a course for testing
+
+        Creates two videos, each with two encodings.
```

Comment:
There should also be a video that has an encoding in one profile but not the other.